### ISSUE TITLE
Doesn't mount anymore

### ISSUE THREAD

**Issue by LeonardoB25** [jstaf/onedriver#428]
Hello, onedriver suddently stopped working today. Even if I toggle the mount option and the boot option and even after reboot, it won't mount the drive anymore :(

**Comment by Dollique** [comment]
I had the same issue. Delete the mount point and remount using the workaround describe here:
https://github.com/jstaf/onedriver/issues/427

**Comment by LeonardoB25** [comment]
yeah that solution always worked for me. I was wondering if i could help to solve the problem at the root cause but i don't think it's that easy

**Comment by girifox** [comment]
Same for me, I'm not a technical Linux user, so I can't work out what I'm supposed to do with the workaround
`WEBKIT_DISABLE_DMABUF_RENDERER=1 onedriver <MOUNT_POINT> 
`

**Comment by girifox** [comment]
I just tried it and it's working for me, just had a delay I guess whilst it was syncing.
